youtube_dl/extractor/regiotv.py

```diff
@@ -3,59 +3,38 @@
 
 from .common import InfoExtractor
 
-from ..utils import (
-    sanitized_Request,
-    xpath_text,
-    xpath_with_ns,
-)
-
 
 class RegioTVIE(InfoExtractor):
```

Comment:
Instead of this extractor you should implement extractor that processes [ViMP platform](https://www.vimp.com/de/) iframe URLs:
http://k100186.vimp.mivitec.net/video/HDC-MAG-quotVirtuelle-Identitaetenquot/a22cd0ac9a14eaaafe0820d66c7971fd
https://www.hd-campus.tv/video/City-of-Colour-Die-Stadt-der-vielen-Gesichter/11/12152f0f79d20e0affe363a2c56f83f1
https://www.schwaebische.de/landkreis/landkreis-ravensburg/ravensburg_video,-erkl%C3%A4rvideo-was-ist-eigentlich-die-kehrwoche-_vidid,138463.html
http://vimp.schwaebische.de/video/Ausschenken-statt-Einschenken/c29897ac7bb59c5c83ad8b64f15e9e69
http://www.web-tv-produktion.de/category/video/Video-zur-Stadtrundfahrt-Stuttgart-mit-dem-Hop-on-Hop-off-Bus-Karrideo-Image-und-Eventfilmproduktion/57e70e43bc958d51d8e2f621f7cdebc8/8
Many more.